files/build_templates/manifest.json.j2

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.0",
```

Comment:
should not this be same as docker version in docker*.mk ?

Reply:
no, this is a manifest version number: https://github.com/Azure/SONiC/blob/16a9357871a6970f8f3dafff85a26ed659561840/doc/sonic-application-extention/sonic-application-extention-hld.md#manifest

Reply:
ok